recipes/kibana/.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
If you say so, I actually don't know what this file is about I simply copied what you did here  4e09d062fae8544f6471d28a099f04047f59ae3b